### PR TITLE
Minor updates

### PR DESCRIPTION
**Pull request by Purg**
These were some recent minor updates I had found in reintroducing myself into various parts of the codebase.

### CODE REVIEW

File: docker/workspace-build/Dockerfile [hunk 1]
```diff
@@ -4,7 +4,7 @@
 # Assumptions:
 # - ROS_DISTRO is defined with the name of a distribution installed under /opt/ros/
 #
-ARG ROS_VERSION
+ARG ROS_VERSION="foxy"
```
Comment (cameron-a-johnson) [inline docker/workspace-build/Dockerfile:7]: Help me recall: what are the reasons we cannot upgrade from foxy? mmdet was one; were there others?

Comment (cameron-a-johnson) [reply docker/workspace-build/Dockerfile:7]: I was wondering if we should revisit this

**Review by cameron-a-johnson — APPROVED** [review]

File: docker/workspace-build/Dockerfile (hunk 1 above)
Comment (Purg) [reply docker/workspace-build/Dockerfile:7]: mmdet/mmpose specifically.

Comment (cameron-a-johnson) [reply docker/workspace-build/Dockerfile:7]: Right. Noted, thanks.

**Comment by josephvanpeltkw** [comment]
@Purg - this looks good to me!

**Pull request merged**